packages/labs/context/tsconfig-legacy.json

```diff
@@ -0,0 +1,33 @@
+// like tsconfig.json, but with experimental decorators enabled
+{
```

Comment:
In reactive element I made a new tsconfig for the standard decorator tests. Do we want to be consistent?